### ISSUE TITLE
Compiling Source from SDK?

### ISSUE THREAD

**Issue by mlptownsend** [gisinternals/buildsystem#205]
I wanted to try compiling GDAL using VS2022 x64 via the sdk from the SDK page:
https://www.gisinternals.com/sdk.php

It says at the top that you need the source from one of the development, stable, or releases sections. But I'm not sure if I'm just blind, but I cannot locate these source packages. When I click on, say, the [x64 VS2022 link](https://www.gisinternals.com/query.html?content=filelist&file=release-1930-x64-gdal-mapserver.zip) I don't see anything that says it's the source, just compiled binaries and headers. The rest are all MSI installers and don't indicate they're source files either. Did they go missing, or is there another step involved?

I tried just cloning a copy of the gdal repo to a gdal folder in the same directory as the SDK download. And I got this error message about FileGDB:
```
 Unable to find "E:\buildsystem\support\FileGDB_API_1_5_1-VS2017\lib\FileGDBAPI.lib". Set FILEGDB_API_DIR in  to the location of the FileGDB SDK
```
The support folder only contains regex-0.12. So I found that particular file on their github page. Then I got an error about ECW, and they seem to have changed their download structure to requiring approval to download the SDK. So I disabled ECW for now. The next error asked me to add 7z to my path. 

The next error has stuck me.
```

**Comment by szekerest** [comment]
Good point, I'll check if the current SDK contains all items for a standalone compilation.
Theoretically you should only need a visual studio command prompt and git to compile GDAL, and type 'nmake rebuild-gdal' in the SDK root directory. The sources are downloaded automatically during the build.

**Comment by mlptownsend** [comment]
Thanks for looking into it!
